pkg/file/msi.go

```diff
@@ -242,9 +242,25 @@ func decodeStrings(dataReader, poolReader io.Reader) ([]string, error) {
 			}
 			return nil, fmt.Errorf("failed to read pool entry: %w", err)
 		}
+		stringEntrySize := int(stringEntry.Size)
+
+		// For string pool entries too long for the size to fit in a single 4-byte entry, we get an 8-byte entry instead,
```

Comment:
```suggestion
		// For string pool entries too long for the size to fit in a single 2-byte entry, we get an 4-byte entry instead,
```

Reply:
I actually need to reword this, because the _entry_ is 4 bytes but the _size_ is two bytes of that. Will handle in a sec.

Reply:
Revised. Let me know if that's sufficiently clear.